contrib/init/systemd/docker.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Docker Application Container Engine
 Documentation=https://docs.docker.com
-After=network.target docker.socket firewalld.service
+After=network-online.target docker.socket firewalld.service
```

Comment:
I tested this modification on my machine. It works as well as before. All containers autostarts correctly. Also I found [this about Wants](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Wants=):
> This is the recommended way to hook start-up of one unit to the start-up of another unit.

So Wants-option also must be added to Unit-file.